SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs

```diff
@@ -147,7 +147,8 @@ public IActionResult ListCasesTest([FromQuery] ListCasesRequest request)
 
             try
             {
-                return Ok(_processDataUsecase.Execute(mosaicId, request.WorkerEmail));
+                return Ok(_processDataUsecase.Execute(mosaicId, request.WorkerEmail, request.FirstName,
+                    request.LastName, request.DateOfBirth.ToString(), request.Postcode));
```

Comment:
perhaps add a null check for request.DateOfBirth before attempting .ToSTring() since the DateOfBirth value can potentially be null

Reply:
Just realised that when passed from the request, the DOB will already be a string, so this conversion didn't need to happen anyway!